lib/logstash/environment.rb

```diff
@@ -76,10 +76,6 @@ def jruby?
       @jruby ||= !!(RUBY_PLATFORM == "java")
     end
 
-    def windows?
-      Gem.win_platform?
-    end
```

Comment:
but please verify what/why anything is required to be in `bootstrap/environment.rb`. this exists for code that need to be executed **before** the complete logstash environment (logstash-core) is loaded